infra/secretsmanager/main.tf

```diff
@@ -0,0 +1,27 @@
+locals {
+  verify_secret_name = "${var.DSS_SECRETS_STORE}/${var.DSS_DEPLOYMENT_STAGE}/${var.DSS_VERIFY_SECRETS_NAME}"
+  tags = "${map(
+  "Name"      , "${var.DSS_INFRA_TAG_SERVICE}-secrets",
+  "owner"     , "${var.DSS_INFRA_TAG_OWNER}",
+  "managedBy" , "terraform",
+  "project"   , "${var.DSS_INFRA_TAG_PROJECT}",
+  "env"       , "${var.DSS_DEPLOYMENT_STAGE}",
+  "service"   , "${var.DSS_INFRA_TAG_SERVICE}"
+  )}"
+}
+
+resource "aws_secretsmanager_secret" "last_verified" {
+  name        = local.verify_secret_name
+  description = "Time of last replica verification"
+  tags        = local.tags
+}
+
+resource "aws_secretsmanager_secret_version" "last_verified_init" {
```

Comment:
The `last_verified` and `last_verified_init` names you are referencing are local names for the AWS resources in terraform. They have different names because they need to be unique within a module. To create the secret, we have a resource `last_verified`. To initialize the value of that secret, we need another resource `last_verified_init`.